### ISSUE TITLE
Crashes on start on OSX

### ISSUE THREAD

**Issue by snotyak** [kwsch/PKHeX#439]
```
System.EntryPointNotFoundException: AddFontMemResourceEx
  at at (wrapper managed-to-native) PKHeX.PKX:AddFontMemResourceEx (intptr,uint,intptr,uint&)
  at PKHeX.PKX.setPKXFont () [0x0003e] in ~/PKHeX/PKHeX/PKM/PKX.cs:588
  at PKHeX.PKX.get_FontFamilies () [0x00012] in ~/PKHeX/PKHeX/PKM/PKX.cs:572
  at PKHeX.PKX.getPKXFont (System.Single size) [0x00001] in ~/PKHeX/PKHeX/PKM/PKX.cs:578
  at PKHeX.Main..ctor () [0x005c7] in ~/PKHeX/PKHeX/MainWindow/Main.cs:105
  at at (wrapper remoting-invoke-with-check) PKHeX.Main:.ctor ()
  at PKHeX.Program.Main () [0x00039] in ~/PKHeX/PKHeX/Program.cs:29
```

Just looking at the stack trace, I think it's failing from:
```
[DllImport("gdi32.dll")]
private static extern IntPtr AddFontMemResourceEx(IntPtr pbFont, uint cbFont, IntPtr pdv, [In] ref uint pcFonts);
```

Is gdi32.dll something I can just add as a reference?

**Comment by kwsch** [comment]
Duplicate with #229

**Comment by snotyak** [comment]
Using the changes on this commit prevents it from crashing immediately for me. 

https://github.com/julroy67/PKHeX/commit/fcc77fcf54f79950656be8d86a03bfe37c65a3f7

I don't know if this fixes the other issues specified in #229.

**Comment by snotyak** [comment]
@kwsch Using the changes on this commit prevents it from crashing immediately for me. 

https://github.com/julroy67/PKHeX/commit/fcc77fcf54f79950656be8d86a03bfe37c65a3f7

I don't know if this fixes the other issues specified in #229 though or if this breaks windows support.

**Comment by kwsch** [comment]
Tested the change on windows, worked. Merged in #440

I guess I just needed another person to say the fix worked to give me enough confidence to merge that change 👍
